en/docs/references/test-your-service-via-postman.md

```diff
@@ -0,0 +1,145 @@
+
+In this step, let's connect a workspace to your service. You can use workspaces in Postman to manage your Postman components for different tasks. In this example, you can create a workspace to have the collections you create for testing the `World Bank Data to COVID-19 Statistics` service. For more information about workspaces, see [Postman Documentation - Using and managing workspaces](https://learning.postman.com/docs/collaborating-in-postman/using-workspaces/managing-workspaces/).
+
+To create a workspace for your service, follow this procedure:
+
+1. In the **Postman** panel that you opened, expand the **Select or create a workspace** field.
+
+    ![Postman panel](../assets/img/postman/postman-right-panel.png){.cInlineImage-half}
+    
+    As shown above, the drop-down list displays the existing workspaces in your Postman account. You can link your service to an existing workspace or create a new one. In this example, let's create a new workspace.
+    
+2. Click **+ Create New**.
+
+3. Enter a name and a description for the new workspace. For this example, let's enter them as follows:
+
+    | **Field**       | **Value**                       |
+    |-----------------|---------------------------------|
+    | **Name**        | `covid-Status`                  |
+    | **Description** | `Test the covid-Status service` |
+    
+4. Click **Create & Link**.
+
+Now you have linked your service to the new workspace named `covid-Status`.
+
+## Step 4: Create a collection
+
+In this step, you will be creating a collection. The collection runner in Postman runs multiple requests in a specified sequence and logs the results. Each collection represents one request. For more information about collections, see [Postman Documentation - Collection Runner](https://learning.postman.com/docs/running-collections/intro-to-collection-runs/)
+
+!!! tip
+    If you linked an existing workspace to the service and that workspace has existing collections, you can skip this step.
+    
+To create a collection, follow this procedure:
+
+1. Under **Collections** in the **Postman** panel, click **+ Add**.
+
+2. Enter a name and a description. For this example, let's enter them as follows:
+
+    | **Field**       | **Value**                             |
+    |-----------------|---------------------------------------|
+    | **Name**        | `invoke-covid-status service`         |
+    | **Description** | `Collection for covid-status service` |
+    
+3. Click **Create**.
+
+As a result, the collection is created and displayed under **Collections** in the **Postman** panel.
+
+## Step 5: Create a request
+
+In this step, let's create a request that you can send via Postman to invoke your service
+
+To create a request, follow this procedure:
+
+1. In the **Postman** panel, click on the **>** icon of the collection that you created. This opens the **Requests** section.
+
+2. Click **+ Add**.
+
+3. Enter a request name (e.g., `covid-status-request-1`)
```

Comment:
To me names are inconsistent. How about having names as:
`COVID Service Workspace`
`COVID Service Test Collection`
`COVID Country Status Request`

Reply:
```suggestion
3. Enter a request name (e.g., `covid-status-request-1`).
```